js/modules.js

```diff
@@ -15,52 +15,79 @@ CORE.create_module("email-nav", function (sb) {
 CORE.create_module("email-catalog", function (sb) {
     var emails;
 
+    var openEmail = function(e) {
+        sb.hide();
+        sb.notify({
+            type : "open-email",
+            data : e.currentTarget.getAttribute("data-email-id")
```

Comment:
I don't know what to do that ID, we need to be able to associate the dom stuff they click on with the data source somehow. If they changed the id here to another valid id, then it would open that email instead, so not a big problem. 

But if they changed it to an invalid id, would cause errors. Maybe should have error checking for valid id